### PR TITLE
COMMERCE-1702 Fix view order URL in approval notification

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

File: commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/asset/CommerceOrderAssetRenderer.java [hunk 1]
```diff
@@ -104,15 +110,18 @@ public String getURLViewInContext(
...
 
+		CommerceChannel commerceChannel =
+			_commerceChannelLocalService.getCommerceChannelByOrderGroupId(
+				_commerceOrder.getGroupId());
+
 		Group group = GroupLocalServiceUtil.getGroup(
-			_commerceOrder.getGroupId());
+			commerceChannel.getSiteGroupId());
 
 		PortletURL portletURL = PortletProviderUtil.getPortletURL(
```
Comment (marco-leo) [inline commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/asset/CommerceOrderAssetRenderer.java:133]: Handle trasmission workflow as well

**Comment by alecsloan** [comment]
@marco-leo requested changes applied, please review.

**Comment by alecsloan** [comment]
force push reflects rebase

**Comment by marco-leo** [comment]
@ethanbustad Mergin, fell free to ask changes

File: commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/asset/CommerceOrderAssetRenderer.java [hunk 2]
```diff
@@ -104,19 +115,46 @@ public String getURLViewInContext(
...
+			orderRenderCommand = "editCommerceOrder";
+		}
+
 		portletURL.setParameter(
-			"mvcRenderCommandName", "viewCommerceOrderDetails");
+			"mvcRenderCommandName", orderRenderCommand);
+
 		portletURL.setParameter(
-			"commerceOrderId",
+		"commerceOrderId",
```
Comment (ethanbustad) [inline commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/asset/CommerceOrderAssetRenderer.java:151]: Whitespace

**Review by ethanbustad — COMMENTED** [review]
Looks like some SF issues were introduced by the force push. Fixing here: https://github.com/liferay/com-liferay-commerce/pull/1398.

**Pull request merged**